SPECS-EXTENDED/rlwrap/rlwrap.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "rlwrap-0.46.1.tar.gz": "2711986a1248f6ac59e2aecf5586205835970040d300a42b4bf8014397e73e37"
```

Comment:
Looks like the sources are not uploaded to the blob store.
I will upload that in the meantime.

Reply:
Thank you.